cmd/admin-revoker/main.go

```diff
@@ -437,18 +458,18 @@ func main() {
 	command := os.Args[1]
 	flagSet := flag.NewFlagSet(command, flag.ContinueOnError)
 	configFile := flagSet.String("config", "", "File path to the configuration file for this service")
-	dryRun := flagSet.Bool("dry-run", true, "")
+	dryRun := flagSet.Bool(
+		"dry-run",
+		true,
+		"true (default): only queries for affected certificates. false: will perform the requested block or revoke action",
```

Comment:
nit: Honestly I'd include the "only works for subcommands foo and bar" text here, too.